### ISSUE TITLE
Line crossing buttons visible

### ISSUE THREAD

**Issue by lmslachciak** [Samsung/TAU#1028]
Buttons -> Contained Button -> Body Area
Observe: Line crossing buttons on the bottom
![image](https://user-images.githubusercontent.com/2246331/80371449-37bda680-8892-11ea-82dc-7f97587021cc.png)

**Comment by lslachciak** [comment]
This is regression after 0a6c08e824f575d1dc33fc458d77fea423f0715c

**Comment by lslachciak** [comment]
@HunseopJeong @TomaszLukawskiSam we need to think more about this issue.
In 0a6c08e box shadow was defined for ui-content area.
But it causes to have line visible over footer in Body Area area example.

**Comment by lslachciak** [comment]
Maybe we should have fix similar to https://github.com/Samsung/TAU/pull/986/commits/fb3795b7e93cbbdad9de35780ddfaa727a15a8ca
for footer?

**Comment by TomaszLukawskiSam** [comment]
Patch has already merged. We can close this issue.

**Comment by lslachciak** [comment]
PR merged, but I would like to ask @HunseopJeong for double check if PR was OK

**Comment by HunseopJeong** [comment]
@lslachciak, @TomaszLukawskiSam,
We need a following code
```
	mask-border: url(images/0_Round_corner/round.svg) 77;
	mask-border-width: 26 * @px_base;
	border-radius: 26 * @px_base;
	box-sizing: border-box;
	box-shadow: 0 0 0 0.25 * @px_base var(--content-area-line-color) inset;
```
to support the round corner case (#28 page in UX guide).
So we need to recover the code and find the another way to fix the issue.

**Comment by lslachciak** [comment]
@HunseopJeong removed code refers to ui-content not to ui-content-area.
Can't we support corner case using this API?

**Comment by TomaszLukawskiSam** [comment]
@HunseopJeong 
Thank you for comment :) . We knew that it was intended but we did not know where exactly. I verified this issue on Android. There is equivalent of `ui-content` and `ui-content-area`.
Below example shows how the `ui-content` crops the `ui-content-area`

![image](https://user-images.githubusercontent.com/29534410/80565471-d52eec80-89f0-11ea-84a1-b858004be291.png)

There is only rounding without border for `ui-content`.

I think we can recover it partially:
```
mask-border: url(images/0_Round_corner/round.svg) 77;
	mask-border-width: 26 * @px_base;
	border-radius: 26 * @px_base;
```
